<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DinaRemasterII</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>                                                (｡◕‿‿◕｡)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="My_Resources.qrc">
    <normaloff>:/game_skull_head_skull_dead_die_death_icon_133746.ico</normaloff>:/game_skull_head_skull_dead_die_death_icon_133746.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="searchStopsAccessibleCheckBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DinaRemasterII</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Accessible</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchStopsPushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DinaRemasterII</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="searchStopsResultCountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchStopsLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>341</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DinaRemasterII</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Search here</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="selectRouteComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Select a line</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectRouteHeaderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DinaRemasterII</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Route:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectTripComboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>¯\_(ツ)_/¯</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectTripHeaderLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DinaRemasterII</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trip:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="searchStopsTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>474</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="My_Resources.qrc"/>
 </resources>
 <connections/>
</ui>
